specs/testing/SkillsMatrixTesting.md

```diff
@@ -0,0 +1,58 @@
+# Automated Testing of Skills
+Defines the short term goals around the automated testing of skills. 
+
+# Goal
+Automate the testing matrix for Bot/Skill interactions. The full testing matrix runs every night. Failures result in emails. 
+
+# Version of the Botbuilder SDK
+1. For each test run, the latest *preview* Bot Builder SDK should be consumed from the MyGet.
+
+2. Alternatly, a specific version of the BotBuilder SDK could be specified via an Azure Devops variable. 
+
+# Testing Matrix
+The following testing matrix needs to be fully tested
```

Comment:
Should V3 js and V3 dotnet be part of the matrix?

Reply:
Here is a full matrix in case it helps (we can start with a reduce set in v0 so we can develop the infrastructure first):
![image](https://user-images.githubusercontent.com/12264946/73874181-01d3cd80-4821-11ea-966a-137cc1c3a4ee.png)

Reply:
My thinking was Crawl, walk run:
* crawl: for "V0" was just C# (Net core 3.1), JS, and Python. 
* walk: richer scenarios. Simple Invoke, Events, etc. 
* run: Net Core 2.1, v3 SDK C#, v3 SDK JS

That is, having the very basics would be a huge first step.

Reply:
sounds good.

Reply:
Fixed Tables. Updates doc now.

Reply:
Updated in the latest version of the doc.